integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -10,7 +10,37 @@ The following diagram shows a high-level overview of the connector's architectur
 
 <img src="./docs/img/influxdb_timestream_connector_lambda_function_arch.png" width=700/>
 
-## Table Schema
+## Table Mapping
+
+### Single-Table Multi-Measure
+
+The following table shows how the connector maps line protocol elements to Timestream for LiveAnalytics record attributes when table mapping is set to single table.
+
+| Line Protocol Element | Timestream Record Attribute |
+|-----------------------|---------------------------|
+| Timestamp             | Time                      |
+| Tags                  | Dimensions                |
+| Fields                | Measures                  |
+| Measurements          | Measure names              |
+
+Single table mapping ingests all line protocol points ingested through the InfluxDB Timestream Connector to the table defined with the `single_table_name` environment variable. The `measure_name` in each Timestream record is derived from the line protocol measurement.
+
+The following example shows the translation of a two line protocol points into a Timestream for LiveAnalytics table, using a Timestamp with second precision and a `single_table_nam` Lambda environment variable configured to `influxdb-measures`:
+
+#### Line Protocol Points
+
+```
+cpu_load_short,host=server01,region=us-west value=0.64,average=1.24, 1725059274
+weather,location=us-midwest,season=summer temperature=82.0,humidity=71.0 1706480990
+```
+
+#### Resulting influxdb-measures Timestream for LiveAnalytics Table
+
+| host     | region  | measure_name     | time                          | value | average | location   | season | temperature | humidity |
+|----------|---------|------------------|-------------------------------|-------|---------|------------|--------|-------------|----------|
+| server01 | us-west | cpu_load_short   | 2024-08-30 23:07:54.000000000 | 0.64  | 1.24    |            |        |             |          |
+|          |         | weather          | 2024-01-22 26:07:33.000000000 | 0.64  | 1.24    | us-midwest | summer | 82.0        | 71.0     |
```

Comment:
I don't see a `value` or `average` measure for the second line protocol point. Should this be changed?

Reply:
Yes, thanks for noticing.